backend/modules/@apostrophecms/layout-column-widget/index.js

```diff
@@ -0,0 +1,20 @@
+export default {
+  fields(self, options) {
+    return {
+      add: {
+        content: {
+          type: 'area',
+          label: 'Main Content',
+          options: {
+            widgets: {
+              '@apostrophecms/rich-text': {},
+              '@apostrophecms/image': {},
+              '@apostrophecms/video': {},
+              '@apostrophecms/layout': {}
```

Comment:
Infinite nesting of layout is not good UX, & the same number of "steps" for sub-layouts is also not a good experience.

I recommend a `nested-layout` subclass for this, with just 6 stops, and then no more layouts nested in that one (expressed via the config of its `nested-layout-column-widget` friend).